corehq/apps/style/static/style/js/hq.helpers.js

```diff
@@ -91,43 +91,3 @@ $.showMessage = function (message, level) {
     $notice.append(message);
     $(".hq-page-header-container").prepend($notice);
 };
```

Comment:
I moved the jquery additions to `hq_extensions.jquery.js` so they're all in one place.  I was having some difficulty with the other ones, so I just kept 'em here for now.